packages/fhir-converter/src/templates/cda/References/DiagnosticReport/assessment.hbs

```diff
@@ -25,10 +25,12 @@
   // -------------------------------------------------------------------------------------------------
 --}}
 {
+    "fullUrl":"urn:uuid:{{ID}}",
```

Comment:
`Bundle entry missing fullUrl"` - added this for all instances of DiagnosticReport

Reply:
I only did this for the DiagnosticReports because thats what was throwing issues but it begs the question if i should do this for all references as well